src/pages/Provider/Provider.scss

```diff
@@ -0,0 +1,97 @@
+.providerHeader {
+  position: relative;
+
+  &__img {
+    max-height: 60vh;
```

Comment:
I've made some changes to do it look more as the design (but I still don't are working in the CSS since I don't know how it should looks in tablet, for example jeje)

![lastscreenshot](https://user-images.githubusercontent.com/66505715/165857403-80c5601d-4a19-4406-a7f9-2598d3fc7059.png)

![lastscreenshot](https://user-images.githubusercontent.com/66505715/165857596-2437eb4c-0395-47a6-865d-081e003dc45f.png)